query/src/servers/http/clickhouse_handler.rs

```diff
@@ -254,10 +254,7 @@ pub async fn clickhouse_handler_get(
             .map_err(InternalServerError)?
             != 0
             && !stmts.is_empty()
-            && stmts.get(0).map_or(false, InterpreterFactoryV2::check)
-            || stmts
-                .get(0)
-                .map_or(false, InterpreterFactoryV2::enable_default))
+            || stmts.get(0).map_or(false, InterpreterFactoryV2::check))
```

Comment:
1. still parse by old parser first?
2.  mysql handler not updated

Reply:
This PR is not ready for review now, I will invite you for reviewing once ready :laughing:

Reply:
> still parse by old parser first?

Yes, we still need to go old parser first, but it will never return an error anymore.

Except `select`, `insert`, `explain` and `delete`, all queries will be parsed as `SeeYouAgain` and handled by the new planner.

> mysql handler not updated

I have updated `use_planner_v2`, is this enough?

Reply:
I think it is enough to pass the current tests.
but can not remove ` onlyif mysql` in tests/logictest/suites/query/subquery.test.  which is a bug to fix.
Maybe we can leave it to another pr.

Reply:
> but can not remove ` onlyif mysql` in tests/logictest/suites/query/subquery.test. which is a bug to fix.

I don't know what's wrong here. Can you raise an issue about it?

Reply:
https://github.com/datafuselabs/databend/issues/6904